.gitignore

```diff
@@ -6,3 +6,5 @@
 _build/
 .*cache/
 pytestdebug.log
+*.pyc
+pyvenv.cfg
```

Comment:
These belong in your personal config, not in the project config, since they're tool specific things. Ignoring .pycs is, pragmatically, a global thing belong in your global or per-user git config rather than a project one.